Massdrop/fluxible-router
# Pull in fix for Safari issuing an extra popstate nav on page load.

This PR pulls in a change that has been submitted, but not yet accepted, to fluxible-router: https://github.com/yahoo/fluxible/pull/351. I also tried an alternate fix, https://github.com/yahoo/fluxible/pull/353, but it did not appear to address the issues that affect us. 

These changes address two open issues:
* https://app.asana.com/0/33626641634273/80239908661505: FTUE doesn't show on Safari when clicking 'Sign Up With Email' button; seen on Safari desktop and mobile; also Chrome iOS
* https://app.asana.com/0/33626641634273/79804799230580: Safari: can load all-communities without logging in

To test:
* In React, pin the fluxible-router module to commit fa7ce51.
* Test the first issue by signing up with an email address, and verifying that you go through the FTUE, instead of being redirected before completing. Try this on Safari desktop and mobile, and Chrome iOS.
* Test the second issue by logging out; then navigating to http://localhost:3000/all-communities on Safari. Verify that you are redirected to the splash page, instead of being able to view the all-communities page.
